testing/python/metafunc.py

```diff
@@ -21,6 +21,7 @@ def function(): pass
         metafunc = self.Metafunc(function)
         assert not metafunc.fixturenames
         repr(metafunc._calls)
+        assert funcargs._format_args(function) == '()'
```

Comment:
please put the tests of the helpers (also below) rather into dedicated test functions.  Tucking it on existing functions like this mixes things too much.